src/main/java/com/inquirer/controller/LoginController.java

```diff
@@ -0,0 +1,50 @@
+package com.inquirer.controller;
+
+
+import com.inquirer.models.User;
+import com.inquirer.services.UserService;
+import com.inquirer.services.impl.UserServiceImpl;
+import org.apache.tiles.TilesContainer;
+import org.apache.tiles.access.TilesAccess;
+import org.apache.tiles.request.Request;
+import org.apache.tiles.request.servlet.ServletApplicationContext;
+import org.apache.tiles.request.servlet.ServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+@Controller
+@RequestMapping("login")
+public class LoginController {
+    @Autowired private UserService userLoginService;
+
+    @RequestMapping(method = RequestMethod.GET)
+    protected String doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
No need to inject HttpServletRequest and HttpServletResponse. Please fix it everywhere.